packages/react-code-editor/src/components/CodeEditor/CodeEditor.tsx

```diff
@@ -518,23 +519,27 @@ export class CodeEditor extends React.Component<CodeEditorProps, CodeEditorState
           const emptyState =
             providedEmptyState ||
             (isUploadEnabled ? (
-              <EmptyState variant={EmptyStateVariant.small}>
+              <EmptyState variant={EmptyStateVariant.sm}>
                 <EmptyStateIcon icon={CodeIcon} />
                 <Title headingLevel="h4" size="lg">
                   {emptyStateTitle}
                 </Title>
                 <EmptyStateBody>{emptyStateBody}</EmptyStateBody>
-                <Button variant="primary" onClick={open}>
-                  {emptyStateButton}
-                </Button>
-                <EmptyStateSecondaryActions>
-                  <Button variant="link" onClick={this.toggleEmptyState}>
-                    {emptyStateLink}
-                  </Button>
-                </EmptyStateSecondaryActions>
+                <EmptyStateFooter>
+                  <EmptyStateActions>
+                    <Button variant="primary" onClick={open}>
+                      {emptyStateButton}
+                    </Button>
+                  </EmptyStateActions>
+                  <EmptyStateActions>
+                    <Button variant="link" onClick={this.toggleEmptyState}>
+                      {emptyStateLink}
+                    </Button>
+                  </EmptyStateActions>
+                </EmptyStateFooter>
               </EmptyState>
             ) : (
-              <EmptyState variant={EmptyStateVariant.small}>
+              <EmptyState variant={EmptyStateVariant.sm}>
                 <EmptyStateIcon icon={CodeIcon} />
                 <Title headingLevel="h4" size="lg">
                   {emptyStateTitle}
```

Comment:
Similar to above comment. Additionally the Button a few lines below this would need to be wrapped in something; depending what Sarah says to another comment it may either be EmptyStatePrimary or EmptyStateActions.